config/manager/config.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   namespace: numaplane-system
 data:
   config.yaml: |
-    clusterName: "CLUSTER_NAME_VALUE" # Replace with the cluster name where numaplane will get deployed.
     timeIntervalSec: 60
     repoCredentials:
     - url: "github.com/numaproj-labs"
```

Comment:
Should we remove this complete data from config map? as this is for testing purpose and not for prod and user should add these value before deploying

Reply:
good point, yes, the repoCredentials shouldn't be in there either. I kind of think we can leave the `timeIntervalSec` in there and just consider that to be the default.

Reply:
okay, Made the changes accordingly